Library/ViewModels/DiscoveryPageViewModelTests.swift

```diff
@@ -31,6 +31,8 @@ internal final class DiscoveryPageViewModelTests: TestCase {
   internal override func setUp() {
     super.setUp()
 
+    self.vm = DiscoveryPageViewModel()
```

Comment:
Moved this here so that the VM is always created after the environment is set up, it was causing the scheduler advancement to fail in the test below.